src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java

```diff
@@ -137,6 +140,24 @@ public DepositManagementSteps() {
             assertThat(account.getBalance()).isEqualByComparingTo(new BigDecimal(balanceWithInterest));
         });
 
+        Given("^there is a customer who is about to open a new deposit of any kind$", () -> {
+            amount = new BigDecimal(1000);
+            account.setBalance(amount);
+        });
+
+        And("^he decided to add the insurance to the deposit$", () -> {
+            interestPolicy = new DailyInterestPolicyWithInsurance(new BigDecimal(10), new BigDecimal(0.05));
```

Comment:
Consider changing scenario to "he decided to add the insurance cost (.+)% to the deposit". This way insurance cost could be passed to this method and changed only in deposit_management.feature. Similar as it is done in Then step.